### DISCUSSION TITLE
Requesting certs using Azure Resource Manager

### DISCUSSION THREAD

**Discussion by mauve** [polymind-inc/acmebot#320] [Q&A] [answered]
Hi,

I would like to integrate keyvault-acmebot with our ARM Bicep deployment somehow, is there are way to specify which certs should be requested using ARM Bicep?

Kind regards,

Mikael

**Comment by shibayan** [comment, marked as answer]
Unfortunately, it is not possible to integrate certificate issuance with Resource Manager at this time.

I'm looking into integration using Custom Provider in Issue #232, but since Custom Provider is in preview and has very limited regions, I would like to look into other ways to do this.

**Reply by mauve** [reply]
if it were possible to specify the certs via AppSettings we might be able to shoe-horn something together.
